### PR TITLE
Advanced settings problem for WMS and CSW services #5932

### PR DESCRIPTION
**Pull request by myrdstom**
## Description
This PR makes it possible to click on the  `Search on service selection` and `Enable localized styles` options from the Advanced Settings for a WMS service and  to click on the `Search on service selection` option for a CSW service

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#5932

**What is the new behavior?**
This PR makes it possible to click on the  `Search on service selection` and `Enable localized styles` options from the Advanced Settings for a WMS service and  to click on the `Search on service selection` option for a CSW service
ezgif com-optimize

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,5 @@ web/docs
 .settings/
 debug.log
 .vscode/settings.json
+.idea/
+yarn.lock
```
Comment (MV88) [inline .gitignore:28]: please remove this and use npm for this project
```suggestion
yarn.lock
```

**Review by MV88 — CHANGES_REQUESTED** [review]
just a quick thing

File: .gitignore (hunk 1 above)
Comment (myrdstom) [reply .gitignore]: done!!

**Review by MV88 — APPROVED** [review]

**Comment by tdipisa** [comment]
@myrdstom please provide a backport to this PR to 2020.02.xx branch for the next minor release.

**Comment by MV88** [comment]
there is no need for the backport

**Pull request merged**